app/src/lib/editors/win32.ts

```diff
@@ -10,663 +10,595 @@ import {
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{15406187-F49E-4822-CAF2-1D25C0C83BA2}',
+      },
+      // 32-bit version of SlickEdit Pro 2017
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\WOW6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{15006187-F49E-4822-CAF2-1D25C0C83BA2}',
+      },
+      // 64-bit version of SlickEdit Pro 2016 (21.0.1)
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{10C06187-F49E-4822-CAF2-1D25C0C83BA2}',
+      },
+      // 64-bit version of SlickEdit Pro 2016 (21.0.0)
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{10406187-F49E-4822-CAF2-1D25C0C83BA2}',
+      },
+      // 64-bit version of SlickEdit Pro 2015 (20.0.3)
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{0DC06187-F49E-4822-CAF2-1D25C0C83BA2}',
+      },
+      // 64-bit version of SlickEdit Pro 2015 (20.0.2)
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{0D406187-F49E-4822-CAF2-1D25C0C83BA2}',
+      },
+      // 64-bit version of SlickEdit Pro 2014 (19.0.2)
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\{7CC0E567-ACD6-41E8-95DA-154CEEDB0A18}',
+      },
+    ],
+    executableShimPath: ['win', 'vs.exe'],
+    expectedInstallationChecker: (displayName, publisher) =>
+      displayName.startsWith('SlickEdit') && publisher === 'SlickEdit Inc.',
+  },
+  {
+    name: 'JetBrains Webstorm',
+    registryKeys: [
+      // Webstorm 2018.3
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\WOW6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\WebStorm 2018.3',
+      },
+      // Webstorm 2019.2
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\WOW6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\WebStorm 2019.2',
+      },
+      // Webstorm 2019.2.4
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\WOW6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\WebStorm 2019.2.4',
+      },
+      // Webstorm 2019.3
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\WOW6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\WebStorm 2019.3',
+      },
+      // Webstorm 2020.1
+      {
+        key: HKEY.HKEY_LOCAL_MACHINE,
+        subKey:
+          'SOFTWARE\\WOW6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\WebStorm 2020.1',
+      },
+    ],
+    executableShimPath: ['bin', 'webstorm.exe'],
+    appInformationExtractor: keys => {
```

Comment:
Since the `expectedInstallationChecker` for this entry already checks whether displayName starts with `WebStorm` we should be able to remove this `appInformationExtractor` method, right?